nautobot_floor_plan/templates/nautobot_floor_plan/floorplan_retrieve.html

```diff
@@ -33,6 +33,10 @@
                 <td>Tile Depth (Relative Units)</td>
                 <td>{{ object.tile_depth }}</td>
             </tr>
+            <tr>
+                <td>Children</td>
```

Comment:
I think we should use the name `Tiles` instead of `Children`.

```suggestion
                <td>Tiles</td>
```

Reply:
I agree, better name for sure.